Sources/SwiftParserDiagnostics/LexerDiagnosticMessages.swift

```diff
@@ -325,6 +325,50 @@ extension SwiftSyntax.TokenDiagnostic {
         changes.append(.replaceLeadingTrivia(token: nextToken, newTrivia: []))
       }
       return [FixIt(message: .removeExtraneousWhitespace, changes: changes)]
+    case .spaceAtStartOfRegexLiteral:
+      var ancestor = Syntax(token)
+      while ancestor.kind != .regexLiteralExpr, let parent = ancestor.parent {
+        ancestor = parent
+      }
+
+      let regexLiteral = ancestor.cast(RegexLiteralExprSyntax.self)
```

Comment:
We should return with `[]` if we can’t find a parent that’s a `regexLiteralExpr` instead of crashing here, just to be safe.